components/Form/GameCardForm.js

```diff
@@ -114,7 +119,14 @@ export default function AddGameCardForm({ appendNewGameCard }) {
           </StyledListItem>
         </StyledList>
       </StyledFieldSetRadioButtons>
-      <AddButton type="submit">Add GameCard</AddButton>
+      <StyledFieldSetButtons>
+        <Link href="/gameplan" passHref>
+          <a>
+            <Button type="cancel">Cancel</Button>
```

Comment:
Why don't you style the anchor tag? Then you wouldn't need to wrap the a-tag around the button?

Reply:
Hey! I did this because I need a button for the submit event and I wanted the "cancel"-link to look exactly like the submit button.

So it was more efficient for me to use the existing button component and just wrap the anchor tag around them for the routing